src/main/java/com/synopsys/integration/jira/common/cloud/configuration/JiraCloudRestConfigBuilder.java

```diff
@@ -64,7 +65,7 @@
     private final BuilderProperties builderProperties;
     private IntLogger logger = new PrintStreamIntLogger(System.out, LogLevel.INFO);
     private Gson gson = new Gson();
-    private AuthenticationSupport authenticationSupport = new AuthenticationSupport();
+    private AuthenticationSupport authenticationSupport = new AuthenticationSupport(new UrlSupport());
```

Comment:
Is it expected to manually pass an empty UrlSupport object each time you instantiate AuthenticationSupport?